app/terms-condition/page.tsx

```diff
@@ -11,52 +13,133 @@ export default function TermsCondition(){
+      </Typography>
+
+      <Typography variant="body1"> 
+      { headsection3.title1 } 
+        <br />
+      { headsection3.content1 } 
+      </Typography>
+<br /> 
+      <Typography variant="body1"> 
+      { subsection6.title2 }
+        <br />
+       { subsection6.content2 } 
+       </Typography>
+<br />
+      <Typography variant="h4"> 
+      { headsection4.subsection7 } 
+      </Typography>
 
-      <h3> 1.2 Changes to Terms: </h3>
-      <p> We reserve the right to modify, update, or change these Terms at any time without prior notice. It is your responsibility to review these Terms periodically to stay informed of any updates. Your continued use of our website or services after any changes to the Terms constitutes your acceptance of the revised Terms.</p>
+      <Typography variant="body1"> 
+      { headsection4.title1 } 
+        < br />
+      { headsection4.content1 } 
+      </Typography>
 
-      <h1> 2. Intellectual Property </h1>
-      <h3> 2.1 Ownership: </h3>
-      <p> The content, design, logos, trademarks, and other intellectual property displayed on our website are owned by SLC or licensed to us. You may not use, reproduce, distribute, or modify any of our intellectual property without our prior written consent. </p>
+<br />
+      <Typography variant="body1"> 
+      { subsection8.title2 } 
+<br />
+      { subsection8.content2 } 
+      </Typography>
+<br />
+      <Typography variant="h4"> 
+      { headsection5.subsection9 } 
+      </Typography>
 
-      <h3> 2.2 User Content: </h3>
-      <p> Any content you submit or contribute to our website, such as comments, reviews, or feedback, remains your intellectual property. However, by submitting such content, you grant us a worldwide, royalty-free, non-exclusive license to use, reproduce, modify, adapt, publish, and distribute your content for the purposes of operating and promoting our website and services. </p> 
+      <Typography variant="body1"> 
+      { headsection5.content } 
+      </Typography>
+<br />
+      <Typography variant="h4"> 
+      { headsection6.subsection10 } 
+      </Typography>
 
-      <h1> 3. Limitations of Liability </h1>
-      <h3> 3.1 Disclaimer of Warranties: </h3>
-      <p> Our website and services are provided on an “as is” and “as available” basis without warranties of any kind, either express or implied. We do not guarantee the accuracy, reliability, or completeness of any information or content on our website. </p>
-  
-      <h3> 3.2 Limitation of Liability: </h3>
-      <p> To the extent permitted by law, we shall not be liable for any direct, indirect, incidental, consequential, or punitive damages arising out of or in connection with your use of our website or services. This includes, but is not limited to, any errors, omissions, or interruptions in the availability of our website or services. </p>
+      <Typography variant="body1"> 
+      { headsection6.content1 } 
+      </Typography>
+<br />
```

Comment:
< br > tags are not necessary for spacing between elements in JSX. Please remove and use alternative MUI elements.